### PR TITLE
Switch scalability to 1.5 release

### PR DESCRIPTION
**Pull request by wojtek-t**
@gmarek

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1083)

### CODE REVIEW

File: jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml [hunk 1]
```diff
@@ -485,39 +487,38 @@
...
+                export REGISTER_MASTER="true"
+                # Reduce logs verbosity
+                export TEST_CLUSTER_LOG_LEVEL="--v=2"
+                # TODO: Remove when we figure out the reason for occasional failures #19048
+                export KUBELET_TEST_LOG_LEVEL="--v=4"
+                # Increase resync period to simulate production
+                export TEST_CLUSTER_RESYNC_PERIOD="--min-resync-period=12h"
+                # Increase delete collection parallelism
+                export TEST_CLUSTER_DELETE_COLLECTION_WORKERS="--delete-collection-workers=16"
+                export KUBE_NODE_OS_DISTRIBUTION="debian"
```
Comment (gmarek) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: @saad-ali - shouldn't we run all tests on GCI?

Comment (wojtek-t) [inline jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml]: To clarify - this is consistent with other suites in 1.5 branch now. The question is whether this is what we want.

**Comment by krousey** [comment]
You are missing the similar changes to the test-grid config.

**Comment by krzyzacy** [comment]
I'm moving things to bootstrap - ref #1075 

If it's just bring the comments back to live I can handle them there.

**Comment by krzyzacy** [comment]
assume you also want to enable gci-gce-scalability-release-1.5 as well?

**Comment by krzyzacy** [comment]
I'll close this one via #1086

**Pull request closed without merging**